### PR TITLE
Featured work default state

### PR DESCRIPTION
**Pull request by tommypoa**
### Feature Description:
- If an artist's current feature work is deleted, a new one from his existing works (if any) will be assigned to be the new featured work
- Previously deleteWork does not update the state of the ArtistProfile – added a line to update the state. Not sure if necessary / intentionally left out.

### Related PRs
None.
### Migrations
None.
### Tests Performed, Edge Cases
Tested on all artists.
### Screenshots

CC: @by-co

### CODE REVIEW

File: app/javascript/components/artists/ArtistProfile.jsx [hunk 1]
```diff
@@ -68,6 +67,27 @@ class ArtistProfile extends React.Component {
...
+    });
+    fetch(APIRoutes.artists.update(this.props.artist.id), {
+      method: 'PUT',
+      body: formData,
+      credentials: 'same-origin',
+      headers: {
+        "X_CSRF-Token": document.getElementsByName("csrf-token")[0].content
+      }
+    }).then((data) => {
+      console.log(this.state.artist.featured_work_id)
```
Comment (by-co) [inline app/javascript/components/artists/ArtistProfile.jsx]: Stray print statement?

File: app/javascript/components/artists/ArtistProfile.jsx [hunk 2]
```diff
@@ -68,6 +67,27 @@ class ArtistProfile extends React.Component {
     window.location = `/works/${work_id}/edit`;
   }
 
+  updateFeatured = (work_id) => {
+    let formData = new FormData();
+    const formKeys = ['featured_work_id'];
+    formKeys.forEach(key => {
```
Comment (by-co) [inline app/javascript/components/artists/ArtistProfile.jsx]: nit: since `formKeys` is only length 1, the `forEach` loop is probably not necessary

File: app/javascript/components/artists/ArtistProfile.jsx [hunk 3]
```diff
@@ -76,7 +96,19 @@ class ArtistProfile extends React.Component {
         "X_CSRF-Token": document.getElementsByName("csrf-token")[0].content
       }
     }).then((data) => {
-      window.location = `/artists/` + this.props.artist.id;
+      let works = this.state.works;
+      delete works[work_id];
+      this.setState({ works: works });
```
Comment (by-co) [inline app/javascript/components/artists/ArtistProfile.jsx]: Since you're refreshing the page with `window.location = ...` after updating the artist, you don't need to update state here! State will be updated with another API call in `componentDidMount()`.

Can also remove line 99 and 100 :)

**Comment by tommypoa** [comment]
Thanks for the comments + suggestions- have reflected them in latest commit!
